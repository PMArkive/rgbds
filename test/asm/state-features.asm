MACRO one
!@#$%^&* /* anything */ :'<,>./? ; goes!
ENDM
CHARMAP "char", 2, 3
DEF string EQUS "four"
DEF variable = 5
DEF constant EQU 6
