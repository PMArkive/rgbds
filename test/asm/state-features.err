warning: Ignoring duplicate feature for option '-s': "equ"
warning: Redundant feature before "all" for option '-s'
