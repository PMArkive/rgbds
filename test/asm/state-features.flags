-s equ,var,equ,all:-
