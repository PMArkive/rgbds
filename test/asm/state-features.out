; File generated by rgbasm

; Numeric constants
def constant equ $6

; Variables
def variable = $5

; String constants
def string equs "four"

; Character maps
newcharmap main
charmap "char", $2, $3

; Macros
macro one
!@#$%^&* /* anything */ :'<,>./? ; goes!
endm
